deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/EnvStages.java

```diff
@@ -168,7 +171,7 @@ public void action(@Context SecurityContext sc,
                        @NotEmpty @QueryParam("actionType") ActionType actionType,
                        @NotEmpty @QueryParam("description") String description) throws Exception {
         EnvironBean envBean = Utils.getEnvStage(environDAO, envName, stageName);
-        authorizer.authorize(sc, new Resource(envBean.getEnv_name(), Resource.Type.ENV), Role.OPERATOR);
+        authorizer.authorize(sc, new Resource(envBean.getEnv_name(), Resource.Type.SYSTEM), Role.OPERATOR);
```

Comment:
needs review

Reply:
what's it about ?

Reply:
looks like you are changing /actions api. Is this intended? Or Am I missing something?

Reply:
Oh, I will move this to the right line, thanks. Friday night change :)